src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="post"
+      data-qa="big"
+      class="search-container"
+    >
+      <img
+        src="./images/Search.svg"
+        alt="search icon search-icon--big"
+        class="search-icon search-icon--big"
+      >
+      <input
```

Comment:
Semantically more right, when every input has own label.
Also in is not necessary creaty img element.
Icon just decor and you can create pseudo element for it for label

Reply:
Label wil be have position: relative
And pseudo-element position: absolute